cmd/http-stats.go

```diff
@@ -190,8 +190,6 @@ func (st *HTTPStats) updateStats(r *http.Request, w *httpResponseRecorder, durat
 			st.successDELETEs.Duration.Add(durationSecs)
 		}
 	}
-	// Increment the prometheus http request count with appropriate label
-	httpRequests.With(prometheus.Labels{"request_type": r.Method}).Inc()
```

Comment:
@nitisht I am not sure how number of requests count is redundant with requests duration, can you elaborate ?

Reply:
@vadmeste the field `minio_http_requests_duration_seconds` (basename) is a [histogram](https://prometheus.io/docs/concepts/metric_types/#histogram) so it has `basename_bucket` , `basename_sum` and `basename_count` fields, where `basename_count` is number of events that have been observed.

The `minio_http_requests_total` also had the same data